prusti-common/src/vir/ast/expr.rs

```diff
@@ -85,6 +85,8 @@ pub enum Const {
     Bool(bool),
     Int(i64),
     BigInt(String),
+    /// Used for encoding function pointers.
+    Dummy,
```

Comment:
This looks really hacky. Can we at least name it `FnPtr` so that is clear what is the purpose of it?

Reply:
I guess I think there might be more types where we just need a dummy value, but yes, for now it can just be `FnPtr`.